script/1_0_2current_en_script/Arcueid/Day 4/04_00_ARC04_6.txt

```diff
@@ -12,10 +12,10 @@ My room is not suitable for protection after all... That's right, we should go t
 <Page4>
 And so, here we are.
 Arcueid flagged down a taxi on the main street, and we took it to the next town over.
-She checked into a room at one of the most expensive hotels in the area. Not only did she rent a room...but the entire top floor as well. It was like she was a celebrity.
+She checked into a room at one of the most expensive hotels in the area. Not only did she rent a room... but the entire top floor as well. It was like she was a celebrity.
 <Page5>
 "Wow, it's so tall!
-Human architecture really is advancing at such a rapid pace.
+Human architecture really is advancing at such a pace.
```

Comment:
keeping rapid is better imo

Reply:
Yeah, I think it got dropped because of the rapid on `rapid progress` on the next line (from 日進月歩). Maybe `advancing at a remarkable pace`?
> ほんとメキメキ向上するわよねー